resources/conf/log_config.toml

```diff
@@ -1,6 +1,6 @@
 # The logging configuration file for control plane
 ######### root Level ########
-logfile = "resources/logs/controlPlane.log"
```

Comment:
Whats the purpose of this file

Reply:
Print the control plane logs to a file

Reply:
I was referring to the log_config.toml file, which is the logging config for the control plane, so thats fine I guess

Reply:
yeah. Just like log4j2 configuration for the filter-core